redi-connect-front/src/pages/front/reset-password/SetNewPassword.tsx

```diff
@@ -71,7 +68,6 @@ export const SetNewPassword = (props: RouteComponentProps<RouteParams>) => {
       try {
         await fetchSaveRedProfile(accessToken)
         console.log('saveprofile')
-        setLoading(false)
```

Comment:
Hah, so (just guessing here) formik has some kind of tracking to be able to know the form is loading? (or, "submitting"?)

Reply:
Not sure, also, however I removed it until we define error/submit handling properly.